lib/Doctrine/ORM/Tools/SchemaValidator.php

```diff
@@ -103,7 +103,7 @@ public function validateClass(ClassMetadataInfo $class)
             if (! class_exists($assoc['targetEntity']) || $cmf->isTransient($assoc['targetEntity'])) {
                 $ce[] = "The target entity '" . $assoc['targetEntity'] . "' specified on " . $class->name . '#' . $fieldName . ' is unknown or not an entity.';
 
-                return $ce;
+                continue;
```

Comment:
Would it be possible to write a test for this? Or is it too involving?

Reply:
Will try to have a look at it this week.